recipe/meta.yaml

```diff
@@ -9,27 +8,30 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ checksum }}
+  sha256: 6e2fb792ca022f900f374f9659fd3e7c530cd4df7d3b7e84be889093b487639f
 
 build:
   number: 0
   script:
     # So Cython regenerates it:
     - rm -rf _line_profiler.c  # [unix]
     - del /s /q _line_profiler.c  # [win]
-    - {{ PYTHON }} setup.py install --single-version-externally-managed --record=record.txt
+    - {{ PYTHON }} setup.py install --old-and-unmanageable
   entry_points:
     - kernprof=kernprof:main
+  # package is required on linux-s390x
+  skip: True  [s390x]
 
 requirements:
   build:
-    - {{ compiler('c') }}
-
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
```

Comment:
**defaults** does not provide `make` for Windows platforms. This should have a selector to indicate that.
```suggestion
    - make      # [unix]
```